website_product_supplier/__openerp__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': "Website Product Supplier",
+    'category': 'Website',
+    'version': '8.0.1.0.0',
+    'depends': [
+        'auth_supplier',
+        'website_portal',
+        'product_manufacturer'
```

Comment:
Please do not use product_manufacturer, user Odoo core functionality: product.supplierinfo

![27-10-2015 12-48-45](https://cloud.githubusercontent.com/assets/8736623/10757151/3a8777b4-7ca9-11e5-8227-f72cdaeea0d3.png)